theme/less/_homepage.less

```diff
@@ -0,0 +1,95 @@
+.homepage div
+{
+  grid-template-areas: 
+    "header header"
+    "main main"
+    "footer footer";
+}
+
+.homepage main article
+{
+  display: grid;
+  grid-template-columns: auto auto;
+}
+
+.homepage main article > :first-child
```

Comment:
This will search for article elements and select first of them. The space before pseudo-element is mandatory here. And the `>` sign I entered for selector not to select other elements first-childs like `section h3` (although there's no other grids inside `article` so this -> `grid-column` particular property I think will not take effect on that childs).

Reply:
Strange that it works, because space doesn't mean anything AFAIK. Then you probably want here is  `.homepage main article > *:first-child` `*` stands for any element.